docs/modules/ROOT/pages/how-tos/_nav.adoc

```diff
@@ -25,6 +25,8 @@
 **** xref:how-tos/testing/integration/rerunning.adoc[Rerunning an integration test]
 **** xref:how-tos/testing/integration/override-snapshots.adoc[Creating an override snapshot]
 **** xref:how-tos/testing/integration/choosing-contexts.adoc[Choosing when to run certain Integration Tests]
+**** Third Parties
+***** xref:how-tos/testing/integration/third-parties/testing-farm.adoc[Testing with Testing Farm]
```

Comment:
Do you anticipate having many more of these? I think that this is the first indentation of this length and if feels like it might be too much, especially since there is no content for "Third Parties."

Reply:
I do anticipate more - how to trigger jenkins, how to trigger prow, etc.